src/templates/components/forms_common_rows.html.tt

```diff
@@ -1,5 +1,28 @@
 [% PROCESS components/forms.html.tt %]
 
+[% BLOCK account_type %]
+        <div class='gr-row form-row'>
+            <div class='gr-1 gr-12-m align-end'>
+                <input type='radio' checked name='account_type' id='account_type_default' value='default'>
+            </div>
+            <div class = 'gr-8 gr-12-m'>
+                <label for='account_type_default'>
+                    [% l('Create a standard real money account to trade on [_1]', website_name) %]
+                </label>
+            </div>
+        </div>
+        <div class='gr-row form-row'>
+            <div class='gr-1 gr-12-m align-end'>
+                <input type='radio' name='account_type' id='account_type_ico' value='ico'>
+            </div>
+            <div class = 'gr-8 gr-12-m'>
+                <label for='account_type_ico'>
+                    [% l('Create an ICO account to take part in our token sale only. <a href="[_1]" target="_blank">Learn more</a>', request.url_for('ico')) %]
```

Comment:
I assume we want to add the noreferrer stuff to target blank link

Reply:
@negarn Internal links are harmless.

Reply:
👍